integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -35,12 +37,12 @@
  */
 public abstract class AbstractFuseFileSystem implements FuseFileSystem {
 
-  private static final Logger LOG = LoggerFactory.getLogger(AbstractFuseFileSystem.class);
-
   static {
-    LibFuse.loadLibrary();
+    LibFuse.loadLibrary(VersionPreference.NO);
   }
 
+  private static final Logger LOG = LoggerFactory.getLogger(AbstractFuseFileSystem.class);
```

Comment:
Others are put at the first line of the class?

Reply:
In the current codebase, static blocks appear everywhere. Is there a standard placement for static blocks?

Reply:
We are following google java convention, but i have found it talking about the ordering.
From https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#1852
https://stackoverflow.com/questions/7530535/java-coding-convention-about-static-method

static blocks put in front of static variables

Reply:
Then my current placement should be right